openadapt/record.py

```diff
@@ -19,10 +19,12 @@
 
 from loguru import logger
 from pynput import keyboard, mouse
+from tqdm import tqdm
 import fire
 import mss.tools
 
 from openadapt import config, crud, utils, window
+from openadapt.thirdparty_customization.my_queue import SynchronizedQueue
```

Comment:
this looks a little long, maybe we can rename to something else.

objects? tools? not sure, `thirdparty_customization` just seems a little off to me. I'm leaning toward naming it `extensions`.

Reply:
I already have extension directory for other PR/issues.

ANy other suggestions ?

Reply:
New COmmit: [a612535](https://github.com/MLDSAI/OpenAdapt/pull/318/commits/a612535aea4f281c431f94f555dadfecfdf3b941)

I have followed the format that we are using:

eg.
Instead of: from openadapt.scrub import scrub_image
Use: from openadapt import scrub and then we call scrub.scrub_image.

Similarly, I have used the same format and styling here out the our case.

But I am open to other suggestion for the directory.

I could just think of thirdparty_customizaiotn as we are customizing the code from the 3rd party imports.

On thinking should we leave to customize_imports ??

Reply:
the `sq` thing looks fine. I think `extensions` makes sense since we're extending a builtin module, but customized_imports does sound better than thirdparty_customization

Reply:
Done: [e2419cc](https://github.com/MLDSAI/OpenAdapt/pull/318/commits/e2419cc6e60910e0968d90cce75c2c47149f9b08)